generator/template.go

```diff
@@ -195,6 +196,129 @@ function getNotifyEntityArrivalSink<T>(notifyCallback: NotifyStreamEntityArrival
     }
   })
 }
+
+type Primitive = string | boolean | number;
+type RequestPayload = Record<string, unknown>;
+type FlattenedRequestPayload = Record<string, Primitive | Array<Primitive>>;
+
+/**
+ * Checks if given value is a plain object
+ * source: https://github.com/char0n/ramda-adjunct/blob/master/src/isPlainObj.js
```

Comment:
Added source link for `isPlainObject`

Reply:
`This copies the logic from xxx`

Reply:
Ah ok! I will modify the comment as per your suggestion 🙇🏼